roles/srv-announcer/vars/main.yml

```diff
@@ -0,0 +1,19 @@
+# NOTE: variables set here are considered to be CONSTANTS due to high precedence
+
+service_name: "{{ role_name }}"
```

Comment:
please prefix all variables with the same prefix, e.g. make this announcer_service_name. Or, for consistency, use `srv_announcer_` as a prefix everywhere.

Reply:
[Done](https://github.com/wireapp/ansible-sft/pull/8/commits/37a1bfb601755e53dcb37343f52b7956a9cd4543)

Reply:
Thanks!